packages/docusaurus-mdx-loader/src/remark/toc/index.ts

```diff
@@ -124,9 +178,21 @@ export default plugin;
 async function createExportNode(name: string, object: any): Promise<MdxjsEsm> {
   const {valueToEstree} = await import('estree-util-value-to-estree');
 
+  const tocObject = object.map((heading: TOCItem | string) => {
+    if (typeof heading === 'string') {
+      const argumentName = heading.replace('...', '');
```

Comment:
I'd prefer to represent this thing as an object rather than a ... prefixed string